.rubocop.yml

```diff
@@ -81,6 +81,16 @@ Style/DotPosition:
 Style/DoubleNegation:
   Enabled: false
 
+Style/SymbolArray:
+  Enabled: false
+
+Style/IndentHeredoc:
+  Enabled: false
+
+Style/MixinGrouping:
```

Comment:
In latest rubocop, this cop detects `include` of rspec matcher as `Module::include`.
This behavior is strange. It would be the issue on rubocop.
So I disabled.

![screen shot 2017-03-27 at 15 05 59](https://cloud.githubusercontent.com/assets/147051/24342962/f0a9988c-12fe-11e7-902b-ad5132c154c4.png)